bundles/framework/printout/view/items/BasicViewSettingsCollapseItem.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { AdditionalSettings } from './AdditionalSettings';
+
+import { PanelHeader } from './PanelHeader';
+
+const BasicViewPanelContent = ({ state, controller }) => {
+    return <AdditionalSettings state={state} controller={controller} />;
+};
+
+BasicViewPanelContent.propTypes = {
+    state: PropTypes.object.isRequired,
+    controller: PropTypes.object
+};
+
+export const getBasicViewCollapseItem = (key, state, controller) => {
```

Comment:
This is mostly a wrapper for AdditionalSettings. We should just rename `AdditionalSettings` to `BasicViewPanelContent` and add `getBasicViewCollapseItem()` to it.

Reply:
Lets use the AdditionalSettings as name and add the getter getAdditionalSettingsCollapseItem () to it and remove the BasicViewSettingsCollapseItem.jsx